bindings/python/libmata/nfa/nfa.pyx

```diff
@@ -750,25 +750,27 @@ cdef class Nfa:
             }
         )
 
-    def is_in_lang(self, vector[Symbol] word):
+    def is_in_lang(self, vector[Symbol] word, use_epsilon = False):
         """Tests if word is in language.
 
         :param vector[Symbol] word: tested word.
+        :param bool use_epsilon: whether the automaton uses epsilon transitions.
         :return: true if word is in language of the NFA.
         """
         run = Run()
         run.thisptr.word = word
-        return self.thisptr.get().is_in_lang(dereference(run.thisptr))
+        return self.thisptr.get().is_in_lang(dereference(run.thisptr), use_epsilon)
 
-    def is_prefix_in_lang(self, vector[Symbol] word):
+    def is_prefix_in_lang(self, vector[Symbol] word, use_epsilon = False):
         """Test if any prefix of the word is in the language.
```

Comment:
I think we should modify the Python interface accordingly as well. In Python, the problem of named parameters is already solved, so we can easily use the bool parameters with default values.